server/src/mikro-orm.config.ts

```diff
@@ -2,6 +2,7 @@ import path from 'path'
 import { AbstractNamingStrategy, NamingStrategy, Options } from '@mikro-orm/core'
 import { TsMorphMetadataProvider } from '@mikro-orm/reflection'
 import { SqlHighlighter } from '@mikro-orm/sql-highlighter'
+require('dotenv').config()
```

Comment:
Looks kind of weird in our otherwise `import`-laden code.

[This website](https://www.npmjs.com/package/dotenv#preload) seems to mention an alternative way?

Reply:
Nice, added in.